src/kernels/miopen_type_traits.hpp

```diff
@@ -26,6 +26,9 @@
 #pragma once
 
 #ifdef MIOPEN_DONT_USE_HIP_RUNTIME_HEADERS
+/// \todo: exact version from which this WA is needed should be updated in release
+/// https://github.com/ROCm/MIOpen/issues/2651
+#define WORKAROUND_USE_MORE_CUSTOM_TYPE_TRAITS (HIP_PACKAGE_VERSION_FLAT > 6001024000ULL)
```

Comment:
@atamazov reviewers be noted that this line **excludes** the assumed (fake) version.